de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/WikiConstants.java

```diff
@@ -30,18 +30,18 @@
     /**
      * Shortcut for System.getProperty("line.separator").
      */
-    public static final String LF = System.getProperty("line.separator");
+    String LF = System.getProperty("line.separator");
```

Comment:
Constants should be `static final`. Is there any specific reason that you made them `package private`?

Reply:
seems, IDE has optimized that away... will fix it with the next cosmetic commit.

Reply:
please not: 
this comment should be a general discussion comment, as it is unrelated to the change commented here.

Reply:
The master job and the PR job used different URLs for the repo (one with the `.git` suffix and one without) - that may have causes Jenkins to not be able to correlate the coverages to each other. I have updated the master job to use the same URL as the PR job - so let's see if this solves the problem with upcoming builds. Mind that master needs to run once before it can take effect.